modules/client/examples/01-client/13-prepare-installation.ts

```diff
@@ -25,7 +25,7 @@ const prepareInstallationParams: PrepareInstallationParams = {
     1234,
```

Comment:
It's a ficticious example of what a custom plugin might need. Depending on what a given plugin does it will vary. Depending on the given ABI, this value will be accepted or rejected.

Reply:
Great. Let's specify this so developers understand what that variable is expected to mean.